PDF/PDFElement.cs

```diff
@@ -291,14 +291,21 @@ public PDFElement()
       int       pageMargin      = PDFConst.DefaultPageMargin,
       float     zoom            = PDFConst.DefaultZoom,
       bool      shouldDisplay   = true,
-      string    subtitle        = null)
+      string    subtitle        = null,
+      double    priority        = -1)
     {
       PDFElement pdfEl;
       string     title;
       string     author;
       string     creationDate;
       string     filePath;
 
+
+      if (priority < 0 || priority > 100)
```

Comment:
Good call. Can you also add a `LogTo.Error(...)` message since this should never happen ? That way a bug report will be sent if it does happen

Reply:
Ok sorted.